File: Week 1 - C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int hash;
    do
    {
        printf("Input an height: ");
        scanf("%d", &hash);
        printf("\n");
        while (getchar() != '\n');
    }
    while (hash <= 0 || hash >= 9);
    int j = 0;
    int i = 0;
    for (i; i < hash; i++)
    {
        for (j = i; j < hash - 1; j++)
        {
            printf(" ");
        }
        for (j = - 1; j < i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (j = - 1; j < i ; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6 - Python/sentimental-mario-more/mario.py
# TODO

from cs50 import get_int

hash = get_int("Height: ")
while (hash <= 0 or hash > 8):
    hash = get_int("Height: ")
for x in range(hash):
    print(" " * (hash-(x+1)) + "#" * (x+1) + "  " + "#" * (x+1))

